Lib/socketserver.py

```diff
@@ -230,7 +230,7 @@ def serve_forever(self, poll_interval=0.5):
 
                 while not self.__shutdown_request:
                     ready = selector.select(poll_interval)
```

Comment:
Why not "if not self.__shutdown_request: break"? Is it useful to call self.service_actions() even if we asked to stop the server? If you add this break, please add a short comment to explain how it can happen and add a "bpo-35017: " prefix to your comment to add the context.

Reply:
You are right. I will make the change right away.